core/templates/cowdata.h

```diff
@@ -133,14 +133,10 @@ class CowData {
 	}
 
 	_FORCE_INLINE_ USize _get_alloc_size(USize p_elements) const {
-		return next_po2(p_elements * sizeof(T));
+		return next_po2(p_elements) * sizeof(T);
 	}
 
 	_FORCE_INLINE_ bool _get_alloc_size_checked(USize p_elements, USize *out) const {
-		if (unlikely(p_elements == 0)) {
-			*out = 0;
-			return true;
-		}
```

Comment:
Why this change? This was done to prevent errors

Reply:
Because if `p_elements = 0`, then `_get_alloc_size` returns 0, because `_next_po2` method has a check for 0. And if you look below, I put `return true;` instead of `return *out;`, because method `_get_alloc_size_checked` returns false in case of overflow.

Reply:
No it doesn't, on anything except 32 bit GNUC it is:
```cpp
_FORCE_INLINE_ bool _get_alloc_size_checked(USize p_elements, USize *out) const {
	*out = _get_alloc_size(p_elements);
	return true;
}
```
No checks at all

Reply:
Make sure this doesn't break when `_get_alloc_size` overflows, that's why this whole thing was added, at minimum the `return *out;` should be restored